dev-tools/mage/gotest.go

```diff
@@ -94,6 +118,33 @@ func DefaultTestBinaryArgs() TestBinaryArgs {
 	}
 }
 
+// GoTestIntegrationForModule executes the Go integration tests.
```

Comment:
Maybe we should mention that tests here are run sequentially, and maybe explain the motivation of it.

Reply:
Good point, done.